custom/custom.css

```diff
@@ -212,6 +212,7 @@ section.cover .cover-main > p:last-child a:last-child {
 
 .markdown-section a {
   font-weight: unset;
+  color: #dcdca9;
```

Comment:
The buttons currently have purple-on-blue which is unreadable
before vs after

![image](https://github.com/vyperlang/learn/assets/2369243/970ee7e1-cb77-41e6-8939-f6ad4b70de07)